scripts/host_cap.py

```diff
@@ -440,6 +467,117 @@ def add_constituent_vars(cap, host_model, suite_list, run_env):
+    >>> tendency_vars = [Var({'local_name':'qv_tend', 'standard_name':'tendency_of_water_vapor', 'units':'kg kg-1 s-1', 'dimensions':'(horizontal_dimension, vertical_layer_dimension)', 'type':'character', 'kind':'len=32'}, _API_SOURCE, _API_DUMMY_RUN_ENV)]
+    >>> check_tendency_variables(tendency_vars, const_vars)
+    ("\\nMismatch tendency variable type 'character' for variable 'tendency_of_water_vapor'. Type should match constituent type of 'real'\\nMismatch tendency variable kind 'len=32' for variable 'tendency_of_water_vapor'. Kind should match constituent kind of 'kind_phys'", 1)
+    >>> tendency_vars = [Var({'local_name':'qv_tend', 'standard_name':'tendency_of_water_vapor_mixing_ratio', 'units':'kg kg-1 s-1', 'dimensions':'(horizontal_dimension, vertical_layer_dimension)', 'type':'real', 'kind':'kind_phys'}, _API_SOURCE, _API_DUMMY_RUN_ENV)]
+    >>> tendency_vars.append(Var({'local_name':'qc_tend', 'standard_name':'tendency_of_cloud_liquid', 'units':'kg kg-1 s-1', 'dimensions':'(horizontal_dimension, vertical_layer_dimension)', 'type':'real', 'kind':'kind_phys'}, _API_SOURCE, _API_DUMMY_RUN_ENV))
+    >>> check_tendency_variables(tendency_vars, const_vars)
+    ("\\nInvalid tendency variable 'tendency_of_water_vapor_mixing_ratio'. All constituent tendency variable standard names must be of the form 'tendency_of_<constituent_stdname>'\\nInvalid tendency variable 'tendency_of_cloud_liquid'. All constituent tendency variable standard names must be of the form 'tendency_of_<constituent_stdname>'", 1)
+    """
+    errflg = 0
+    errmsg = ''
+
+    for tend_var in tend_vars:
+        prop_error = False
+        valid_var = False
+        tend_stdname = tend_var.get_prop_value('standard_name')
+        tend_units = tend_var.get_prop_value('units')
+        tend_stdname_split = tend_stdname.split('tendency_of_')[1]
+        tend_dimensions = tend_var.get_dimensions()
+        tend_type = tend_var.get_prop_value('type')
+        tend_kind = tend_var.get_prop_value('kind')
+        for const_var in const_vars:
+            const_stdname = const_var.get_prop_value('standard_name')
+            const_units = const_var.get_prop_value('units')
+            const_dimensions = const_var.get_dimensions()
+            const_type = const_var.get_prop_value('type')
+            const_kind = const_var.get_prop_value('kind')
+            if tend_stdname_split == const_stdname:
+                # We found a match! Check units
+                if tend_units.split('s-1')[0].strip() != const_units:
+                    errmsg += f"\nMismatch tendency variable units '{tend_units}'"
+                    errmsg += f" for variable '{tend_stdname}'."
+                    errmsg += f" No variable transforms supported for constituent tendencies."
```

Comment:
```suggestion
                    errmsg += " No variable transforms supported for constituent tendencies."
```

Since this line has no string substitutions

Reply:
'f' removed!